### PR TITLE
🐛 Fix string parsing issue when submit ISCN

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: likecoin/admin/views/metabox.php [hunk 1]
```diff
@@ -199,6 +199,7 @@ function likecoin_add_meta_box( $post, $button_params, $publish_params ) {
 				'postId'  => $post_id,
 			)
 		);
+
```
Comment (williamchong) [inline likecoin/admin/views/metabox.php:202]: Please remove unused newline

File: likecoin/admin/matters.php [hunk 2]
```diff
@@ -166,11 +166,13 @@ function likecoin_handle_matters_api_error( $error ) {
  * @param WP_Post| $post Post object.
  */
 function likecoin_get_post_tags_for_matters( $post ) {
-	$post_id   = $post->ID;
-	$func      = function( $terms ) {
-		return $terms->name;
+	$post_id              = $post->ID;
+	$raw_encode_post_tags = array();
```
Comment (williamchong) [inline likecoin/admin/matters.php:170]: Please remove `$raw_encode_post_tags` if it is not used

**Review by williamchong — CHANGES_REQUESTED** [review]

File: likecoin/admin/views/metabox.php (hunk 1 above)
Comment (shellyYG) [reply likecoin/admin/views/metabox.php:202]: ok!

File: likecoin/admin/matters.php (hunk 2 above)
Comment (shellyYG) [reply likecoin/admin/matters.php:170]: ok!

**Pull request merged**